jest.config.js

```diff
@@ -6,7 +6,7 @@ process.env.TZ = 'UTC';
 
 module.exports = {
     preset: 'jest-expo',
-    verbose: true,
+    verbose: false,
```

Comment:
Too verbose causing the leaks?

Reply:
No. Too verbose was making so github took a lot of time to show the CI results, because the logs were too long. We don't need that kind of verbosity, and now it loads the test results in seconds.

Again, a minor fix piggybacked to the leaks.